tests/tests/integration_tests.rs

```diff
@@ -439,6 +517,35 @@ async fn test_eth_api(ctx: TestContext) -> anyhow::Result<()> {
     Ok(())
 }
 
+async fn subgraph_data_sources(ctx: TestContext) -> anyhow::Result<()> {
+    let subgraph = ctx.subgraph;
+    assert!(subgraph.healthy);
+    let expected_response = json!({
+        "mirrorBlocks": [
+            {  "number": "0" },
+            {  "number": "1" },
+            { "number": "2" },
+            {  "number": "3" },
+            {  "number": "4" },
+            {  "number": "5" },
+            { "number": "6" },
+            {  "number": "7" },
+            {  "number": "8" },
+            {  "number": "9" },
```

Comment:
The formatting here is off, and #5599 needs to correct that. Would be better to adjust the formatting here

Reply:
It's again changed in [#5602](https://github.com/graphprotocol/graph-node/pull/5602), so neither is actually needed.